about-page.html

```diff
@@ -0,0 +1,267 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <title>Document</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css"
+    />
+    <link rel="stylesheet" href="about-page.css" />
+  </head>
+  <body>
+    <!-- nav banner starts -->
+    <section class="nav-banner">
+      <ul class="nav-banner-list">
+        <li class="nav-banner-item">
+          <img src="./new-img/facebook.png" alt="facebook-logo" />
+        </li>
+        <li class="nav-banner-item">
+          <img src="./new-img/twitter.png" alt="" />
+        </li>
+        <li class="nav-banner-item">English</li>
+        <li class="nav-banner-item">My Page</li>
+        <li class="nav-banner-item">Logout</li>
+      </ul>
+    </section>
```

Comment:
- [OPTIONAL] I would advise you to change the top-level `section` tag with the `nav` tag for semantic reasons. The `nav` tag usually contains navigation links and links to other external sites. 👍 😎